docs/api/textarea.md

```diff
@@ -29,6 +29,40 @@ import BasicPlayground from '@site/static/usage/v7/textarea/basic/index.md';
 
 <BasicPlayground />
 
+## Label Positioning
+
+Labels will take up the width of their content by default. This positioning can be changed to be a fixed width, stacked, or floating label.
+
+import Labels from '@site/static/usage/v7/textarea/labels/index.md';
+
+<Labels />
```

Comment:
Can we rename the directory `label-placement`? (See previous comment on consistency)